### PR TITLE
Reverting CI code to build sperate debian exclusively for berkeley migration package

### PR DESCRIPTION
**Pull request by dkijania**
Reason to revert the code is discussed in comment for base branch:

```
I still wonder if we want to throw back migration apps to archive node debian. Previously the reason to put berkeley migration to separate debian was different profile. However, we learned interesting things from UMT event, like people are using single machine for migration as well as mainnet archive node. As a result they can have, at one point in time below debian setup:

mainnet archive debian
mainnet maintenance debian package
migration debian package (after this change with archive node)
Which must leads to a conflict as we are trying to install more than one replayer and archive node debian. IMHO we should revert out approach to have separate berkeley-migration package with only:

mina-berkeley-migration
mina-migration-repplayer
mina-berkeley-migration-verifier
berkeley-migration.sh script
```

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Review by emberian — APPROVED** [review]

**Comment by deepthiskumar** [comment]
!ci-build-me

File: scripts/deb-builder-helpers.sh [hunk 1]
```diff
@@ -353,16 +354,15 @@ build_archive_migration_deb () {
...
 
-
   create_control_file "$ARCHIVE_MIGRATION_DEB" "${ARCHIVE_DEPS}" 'Mina Archive Process
  Compatible with Mina Daemon'
 
   cp ./default/src/app/berkeley_migration/berkeley_migration.exe "${BUILDDIR}/usr/local/bin/mina-berkeley-migration"
   cp ./default/src/app/berkeley_migration_verifier/berkeley_migration_verifier.exe "${BUILDDIR}/usr/local/bin/mina-berkeley-migration-verifier"
   cp ./default/src/app/replayer/replayer.exe "${BUILDDIR}/usr/local/bin/mina-migration-replayer"
-
+  cp ./default/scripts/archive/migration/berkeley_migration.sh "${BUILDDIR}/usr/local/bin/mina-berkeley-migration-script"
```
Comment (emberian) [inline scripts/deb-builder-helpers.sh:363]:
```suggestion
  cp ../scripts/archive/migration/berkeley_migration.sh "${BUILDDIR}/usr/local/bin/mina-berkeley-migration-script"
```

**Comment by emberian** [comment]
!ci-build-me

**Comment by emberian** [comment]
https://github.com/MinaProtocol/mina/pull/15405

**Comment by SanabriaRusso** [comment]
#15404 partially resolves the merge conflict in develop. I have added comments to allow it to run in CI @emberian

**Comment by dkijania** [comment]
!ci-nightly-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by SanabriaRusso** [comment]
Even after adding the images referenced in the comment above, the test now fails for a different dependency issue (https://github.com/MinaProtocol/mina/pull/15405)

**Review by SanabriaRusso — APPROVED** [review]
Merging conflicts with `develop` are to be addressed here: https://github.com/MinaProtocol/mina/pull/15405 which is currently failing.

**Comment by dkijania** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!approved-for-mainnet

**Comment by deepthiskumar** [comment]
https://github.com/MinaProtocol/mina/pull/15405 will fix the develop conflict

**Pull request merged**